CHANGELOG.md

```diff
@@ -21,6 +22,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Added `summary` method to Profilers. ([#1259](https://github.com/PyTorchLightning/pytorch-lightning/pull/1259))
 - Added informative errors if user defined dataloader has zero length ([#1280](https://github.com/PyTorchLightning/pytorch-lightning/pull/1280))
 - Added testing for python 3.8 ([#915](https://github.com/PyTorchLightning/pytorch-lightning/pull/915)) 
+- Added support for `IterableDataset` when `val_check_interval=1.0` (default), this will trigger validation at the end of each epoch. ([#1283](https://github.com/PyTorchLightning/pytorch-lightning/pull/1283))
```

Comment:
if you do a rebase, these unrelated changes will disappear :)

Reply:
Now I did the rabase, but there are a lot of commits have appeared (see below). I think, I need to go to read rebase tutorial :)

Reply:
Hmm, I didn't mean to make you more work, sorry :) I don't know what happened, but the good thing is you can always checkout the commit before you did the rebase, and then force push. This will revert it again to what you had before.

Reply:
https://github.com/PyTorchLightning/pytorch-lightning/pull/1278#issuecomment-609016922